### PR TITLE
Update docs (Python 3 is actually supported)

### PR DESCRIPTION
**Pull request by pietrodn**
Update installation instructions and improve suggestions about Python environment management.

### CODE REVIEW

**Comment by justinclift** [comment]
As a compliment to this, the "Hello Drone" source on the [Quick Start](https://dronekit-python.readthedocs.io/en/latest/guide/quick_start.html) page should probably be adjusted to work with Python 3 as well.

Looks like it just needs the `print` statements turned into `print()`.

File: docs/develop/installation.rst [hunk 1]
```diff
@@ -16,23 +16,25 @@ It is installed from **pip** on all platforms:
...
 
-* Mac and Linux require you prefix the command with ``sudo``.
+* Install `dronekit` with `pip` inside a virtualenv:
     
   .. code-block:: bash
 
-      sudo pip install dronekit
+      python3 -m venv .venv
+      . .venv/bin/activate
+      pip install dronekit
```
Comment (peterbarker) [inline docs/develop/installation.rst:25]: You've specified a version for Python but not for pip.  Probably want to be consistent.

File: docs/develop/installation.rst [hunk 2]
```diff
@@ -16,23 +16,25 @@ It is installed from **pip** on all platforms:
...
       
 * On Linux you may need to first install **pip** and **python-dev**:
     
   .. code-block:: bash
 
       sudo apt-get install python-pip python-dev
       
-  Alternatively, the *easy_install* tool provides another way to install pip:
+  Alternatively, you can use the `ensurepip` module to install or upgrade Pip on Python 3:
```
Comment (peterbarker) [inline docs/develop/installation.rst:33]: So does this not work on python2?  Should we leave the easy_install stuff in that case?

File: docs/develop/installation.rst (hunk 1 above)
Comment (pietrodn) [reply docs/develop/installation.rst:25]: Once the virtualenv is activated, `pip` and `pip3` always point to the version of `pip` inside the virtualenv.

File: docs/develop/installation.rst (hunk 2 above)
Comment (pietrodn) [reply docs/develop/installation.rst:33]: Actually it's supported also for Python 2.7: https://docs.python.org/2.7/library/ensurepip.html
I'll change the line

Comment (pietrodn) [inline docs/develop/installation.rst:33]:
```suggestion
  Alternatively, you can use the `ensurepip` module to install or upgrade Pip on your system:

```

**Comment by peterbarker** [comment]
Merged, thanks!

**Comment by justinclift** [comment]
@peterbarker The docs don't seem to be showing this change:

&nbsp; &nbsp; https://dronekit-python.readthedocs.io/en/latest/develop/installation.html

Does something need to be triggered, for that to happen? :smile:

**Comment by peterbarker** [comment]
On Fri, 3 Jul 2020, Justin Clift wrote:

> 
> @peterbarker The docs don't seem to be showing this change:
> 
>     https://dronekit-python.readthedocs.io/en/latest/develop/installation.html
> 
> Does something need to be triggered, for that to happen? 😄

No idea.  Possibly we have to have CI passing for that to happen :-)

I didn't look too closely at why CI is currently failing - it wasn't 
related to this change, AFAICS.

**Comment by justinclift** [comment]
@peterbarker Any idea who needs to be pinged to figure out (for sure) what's needed for the website to be updated? :smile:

Sorry to be persistent, but this PR is kind of important to get published. :wink:

**Comment by peterbarker** [comment]
On Fri, 3 Jul 2020, Justin Clift wrote:

> @peterbarker Any idea who needs to be pinged to figure out (for sure) what's needed for the website to be updated? 😄

I'd definitely look at any PR which was created to fix CI :-)

**Comment by justinclift** [comment]
@peterbarker Sorry, can't help with that.

Just tried to look at the CircleCI piece, but it refuses to even show me the details without authorising a GitHub app.  And **that** insists on permission to all **private** repos I have access to, which I can't allow.

Looks like GitHub **still** doen't have their access permissions story worked out.  Frustrating. :frowning:

**Comment by pietrodn** [comment]
@peterbarker I opened an issue for the failing tests https://github.com/dronekit/dronekit-python/issues/1041

**Pull request merged**